tests/test_story.py

```diff
@@ -18,28 +18,55 @@
 
 import pytest
 
-from telegram import Story
+from telegram import Chat, Story
 from tests.auxil.slots import mro_slots
 
 
 @pytest.fixture(scope="module")
 def story():
-    return Story()
+    return Story(TestStoryBase.chat, TestStoryBase.id)
+
+
+class TestStoryBase:
+    chat = Chat(1, "")
+    id = 0
 
 
 class TestStoryWithoutRequest:
```

Comment:
```suggestion
class TestStoryWithoutRequest(TestStoryBase):
```
makes it easier to reference it (`self`)